game/campaignloader/mizcampaignloader.py

```diff
@@ -351,119 +352,101 @@ def objective_info(
     def add_preset_locations(self) -> None:
         for static in self.offshore_strike_targets:
             closest, distance = self.objective_info(static)
-            closest.preset_locations.offshore_strike_locations.append(
-                PointWithHeading.from_point(
-                    static.position, Heading.from_degrees(static.units[0].heading)
-                )
+            closest.preset_locations.buildings.append(
```

Comment:
I removed the different types because i introduced the `PresetLocation` class which allows differentiation based on the tasks. This makes the whole mizcampaignloader and start_generator code a bit lighter. They could need some love to make them better as i only touched a small part.
This particular list `offshore_strike_locations` was only used for one purpose in the start_generator: Iterate through them an pick "oil" category from the building_set. As this is not needed with the new Task `OffShoreStrikeTarget `and the Role "Building" i removed all the redundant lists.
I personally do like it much more that way as a OffShoreStrikeTarget will just use a "Building" Template. Thats why i merged all building locations to one list.

Reply:
The idea is not store information in 10 lists which just have different names. Use one list and use objects with differentiation from attributes. I find this way cleaner and easier to handle :) But i'm willing to change that back if this is requested :)